schema.graphql

```diff
@@ -80,7 +80,6 @@ type User @entity {
   createdAt: BigInt!
   updatedAt: BigInt!
   platform: Platform # Platform on which user nft was minted
-  numReviews: BigInt! # number of reviews user has received
```

Comment:
@quent043 breaking changes issue here: 
- when this graph update will be deploy, indie will stop working
- we need an indie PR link to that as this value is used
In general for the next PRs we will prevent any breaking changes like this, delete of a field will be not authorized anymore

Reply:
https://github.com/TalentLayer-Labs/indie-frontend/pull/69